core-feature-pack/src/main/resources/content/bin/common.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/sh -x
 
+setJava9_plus() {
+  $JAVA --add-modules=java.se -version > /dev/null 2>&1 && JAVA_9_PLUS=true || JAVA_9_PLUS=false
```

Comment:
We could probably just use `MODULAR_JDK` here as well and maybe change the function name.

Reply:
@jamezp see my comment above about splitting this function into two parts